1.0/debian/runtime-deps/hooks/post_push

```diff
@@ -2,47 +2,35 @@
 
 set -e
 
-# Issue - https://github.com/docker/hub-feedback/issues/384#issuecomment-214508878
-# Docker Hub emits an invalid .dockercfg.  Remove the outer auths element.
-if [[ $DOCKERCFG == {\"auths\":* ]]; then
-    jsonStartIndex=$(expr index "$DOCKERCFG" :)
-    length=$((${#DOCKERCFG} - $jsonStartIndex - 1))
-    echo ${DOCKERCFG:jsonStartIndex:length} > /root/.dockercfg
-fi
-
-
-echo "Pushing runtime-deps images"
-tagStartIndex=$(expr index "$IMAGE_NAME" :)
-repoName=${IMAGE_NAME:0:tagStartIndex - 1}
-
-imageName=$repoName":1.0.3-runtime-deps"
+echo "Pushing additional runtime-deps tags"
+imageName=$DOCKER_REPO":1.0.3-runtime-deps"
 docker tag $IMAGE_NAME $imageName
 docker push $imageName
 
 # Existing core-deps tags are obsolete but being preserved for through the 1* release
-imageName=$repoName":1.0.3-core-deps"
+imageName=$DOCKER_REPO":1.0.3-core-deps"
 docker tag $IMAGE_NAME $imageName
 docker push $imageName
 
-imageName=$repoName":1.0-core-deps"
+imageName=$DOCKER_REPO":1.0-core-deps"
 docker tag $IMAGE_NAME $imageName
 docker push $imageName
 
 
-echo "Pushing runtime images"
-imageName=$repoName":1.0.3-runtime"
+echo "Pushing additional runtime tags"
```

Comment:
I think I'd prefer this to say "Pushing runtime image" because without this hook, no runtime images are on Docker Hub.  "Pushing additional runtime tags" implies to me that the runtime image is already there, which is the case for runtime-deps but not runtime.  This applies to the 1.1 post_push file also.